src/components/Button/Button.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './Button.css';
+import { Props } from './types';
+
+const Button: Props = (props) => {
+    const {
+        children,
+        ...buttonProps
+    } = props;
```

Comment:
Можно сделать деструктуризацию прямо в аргументе функции

```suggestion
const Button: Props = ({children, ...props}) => {
```